includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -169,6 +172,7 @@ public function register() {
 			}
 		}
 
+		add_action( 'enqueue_block_editor_assets', $this->get_method_proxy( 'enqueue_assets_for_non_sitekit_user_on_block_editor_page' ), 40 );
```

Comment:
I believe we should only hook this action when the `rrmModuleV2` feature flag is enabled.